src/OpenApi/Serializer/OpenApiNormalizer.php

```diff
@@ -26,6 +26,8 @@
 final class OpenApiNormalizer implements NormalizerInterface, CacheableSupportsMethodInterface
 {
     public const FORMAT = 'json';
+    public const JSONOPENAPI_FORMAT = 'jsonopenapi';
+    public const YAMLOPENAPI = 'yamlopenapi';
```

Comment:
```suggestion
    public const YAML_FORMAT = 'yamlopenapi';
```

Same, and `FORMAT` suffix is missing